client/packages/common/src/ui/components/panels/StatusChip.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
I've moved the status cell component into panels. Wasn't quite sure where best to put it. But have moved it out of the cell component to expose just the chip.